schema/mysql/v57/temporal/versioned/v1.1/mutable_cluster_metadata.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE cluster_metadata ADD mutable_data BLOB;
+ALTER TABLE cluster_metadata ADD mutable_data_encoding VARCHAR(16);
```

Comment:
ALTER TABLE cluster_metadata ADD mutable_data BLOB  NOT NULL;
ALTER TABLE cluster_metadata ADD mutable_data_encoding VARCHAR(16) NOT NULL;